src/app/dashboard/Missions/MapView.jsx

```diff
@@ -121,8 +121,7 @@ const Overview = ({ currentMission, missions, org, setSelectedMission, volunteer
       SpecificDetails = (
         <>
           <b>Food Box</b>
-          {mission.missionDetails?.needs?.map((box, index) => {
-            //{_.get(mission.missionDetails, "needs")?.map((box, index) => {
+          {_.get(mission.missionDetails, "needs")?.map((box, index) => {
```

Comment:
I thought we stopped using lodash for this pattern

Reply:
Ah yes, this should be removed now.